From 49894107101c6c4549a5eddb9581ff10d0503fe1 Mon Sep 17 00:00:00 2001
From: Mahesh Bodapati <mbodapat@xilinx.com>
Date: Wed, 11 Jan 2017 15:46:28 +0530
Subject: [PATCH 03/51] For dejagnu static testing on qemu, suppress warnings
 about multiple definitions from the test function and libc in line with
 method used by powerpc. Dynamic linking and using a qemu binary which
 understands sysroot resolves all test failures with builtins

Signed-off-by: David Holsgrove <david.holsgrove@xilinx.com>

Upstream-Status: Pending
---
 gcc/testsuite/gcc.c-torture/execute/builtins/builtins.exp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gcc/testsuite/gcc.c-torture/execute/builtins/builtins.exp b/gcc/testsuite/gcc.c-torture/execute/builtins/builtins.exp
index 3d7cdaf5a0c..e0bcdd82274 100644
--- a/gcc/testsuite/gcc.c-torture/execute/builtins/builtins.exp
+++ b/gcc/testsuite/gcc.c-torture/execute/builtins/builtins.exp
@@ -48,14 +48,10 @@ if { [istarget *-*-eabi*]
    lappend additional_flags "-Wl,--allow-multiple-definition"
 }
 
-<<<<<<< HEAD
-=======
 if [istarget "microblaze*-*-linux*"] {
    lappend additional_flags "-Wl,-zmuldefs"
-   lappend additional_flags "-fPIC"
 }
 
->>>>>>> 6ef6e5b... [LOCAL]: Testsuite - builtins tests require fpic
 foreach src [lsort [find $srcdir/$subdir *.c]] {
     if {![string match *-lib.c $src] && [runtest_file_p $runtests $src]} {
 	c-torture-execute [list $src \
-- 
2.43.7

